test/run_tests.sh

```diff
@@ -40,13 +40,14 @@ function run_dynamic {
 }
 
 function run_all_tests {
+  run_dynamic python3 "$CDIR/../../test/test_torch.py" "$@" -v TestViewOpsXLA
   python3 "$CDIR/../../test/test_torch.py" "$@" -v TestTorchDeviceTypeXLA
   run_dynamic python3 "$CDIR/../../test/test_torch.py" "$@" -v TestDevicePrecisionXLA
   python3 "$CDIR/../../test/test_torch.py" "$@" -v TestTensorDeviceOpsXLA
   python3 "$CDIR/../../test/test_indexing.py" "$@" -v TestIndexingXLA
   python3 "$CDIR/../../test/test_indexing.py" "$@" -v NumpyTestsXLA
-  run_dynamic python3 "$CDIR/../../test/test_nn.py" "$@" -v TestNNDeviceTypeXLA
```

Comment:
Why this change?

Reply:
This test fails on our DevFairs so I moved it after TestTypePromotion so we could run more of the test suite easily.

Reply:
I don't understand. If tests fail we should be fixing them or blacklisting them.
This change does not seem to below to this PR.